etc/profile-m-z/rtv-addons.inc

```diff
@@ -0,0 +1,22 @@
+# This file is overwritten during software install.
```

Comment:
`*.inc` files should be placed in `etc/inc`.

Reply:
https://github.com/netblue30/firejail/blob/9b56dc8e94cf2313baf19e5847a0631231497c93/Makefile.in#L135

Reply:
ok